daemon/mgr/network.go

```diff
@@ -182,7 +182,7 @@ func (nm *NetworkManager) Remove(ctx context.Context, name string) error {
 	nw, err := nm.controller.NetworkByName(name)
 	if err != nil {
 		if err == libnetwork.ErrNoSuchNetwork(name) {
-			return errors.Wrap(errtypes.ErrNotfound, err.Error())
```

Comment:
I think we can keep it the wrap here 😄

Reply:
Never ever directly return the ErrNotFound, it will be so ambiguous for user to understand.

No one understands the `not found` if no context is there. @rudyfly